boards/arm/nucleo_l452re/nucleo_l452re_p.dts

```diff
@@ -7,6 +7,7 @@
 
 /dts-v1/;
 #include "nucleo_l452re_common.dtsi"
+#include <st/l4/stm32l452retxp-pinctrl.dtsi>
 #include "arduino_r3_connector.dtsi"
```

Comment:
Same here, might be best placed in nucleo_l452re_common.dtsi

Reply:
Just noticed the 2 inclusions are different... my bad, because revision P has different pinctrl definitions ! I remove my comments !